Bottom: 41275ebf36f106cedef6963331ebce848338db3a
Top:    c271dda29a135a715e69d0061f0e7a93c9215f4c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-17 11:46:00 +0100

Refresh of lesstif-gtk-hids-ignore-iso-le

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index d969dd2..5f5f16e 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -345,6 +345,7 @@ ghid_port_key_press_cb (GtkWidget * drawing_area,
     case GDK_Shift_L:
     case GDK_Shift_R:
     case GDK_Shift_Lock:
+    case GDK_ISO_Level3_Shift:
       break;
 
     case GDK_Up:
diff --git a/src/hid/lesstif/menu.c b/src/hid/lesstif/menu.c
index 297768b..8de2952 100644
--- a/src/hid/lesstif/menu.c
+++ b/src/hid/lesstif/menu.c
@@ -1140,6 +1140,7 @@ lesstif_key_event (XKeyEvent * e)
     case XK_Super_R:
     case XK_Hyper_L:
     case XK_Hyper_R:
+    case XK_ISO_Level3_Shift:
       return 1;
     }
